astroquery/mast/cutouts.py

```diff
@@ -589,12 +633,24 @@ def download_cutouts(self, coordinates, *, size=5, survey=None, cutout_format="f
             The Column Name is the keyword, with the argument being one or more acceptable
             values for that parameter, except for fields with a float datatype where the
             argument should be in the form [minVal, maxVal].
+        timeout_add : int or float, optional
+            The amount (in seconds) by which the request processing time upper limit will be changed.
+            The request processing time by default is 600 seconds, meaning an attempt at communicating
+            with the API will take 600 seconds before timing out. The timeout upper limit can be modified
+            using this argument for large cutout requests via TESSCut. Default is None.
 
         Returns
         -------
         response : `~astropy.table.Table`
             Cutout file(s) for given coordinates
         """
+
+        # Modify TIMEOUT attribute if necessary (usually this is modified for large requests)
+        if timeout_add:
+            self._service_api_connection.TIMEOUT = self._service_api_connection.TIMEOUT + timeout_add
+            log.info(f"Request timeout upper limit is being changed to {self._service_api_connection.TIMEOUT}"
+                     " seconds.")
```

Comment:
I don't think this is a good approach because it permanently modifies the timeout, which is very unexpected. There already exists a configuration parameter for the timeout: https://github.com/astropy/astroquery/blob/770949eda7883fef82c3be714b1f618cdce7d05c/astroquery/mast/__init__.py#L23-L25
It should be simple to use a different value for one function call without modifying it permanently, see https://docs.astropy.org/en/stable/config/index.html#usage-tips